tastytrade/instruments.py

```diff
@@ -0,0 +1,65 @@
+import json
+from dataclasses import dataclass
+from typing import List, Optional
+
+import requests
+
+from tastytrade.session import Session
+from tastytrade.utils import validate_response
+
+
+@dataclass
+class DestinationVenueSymbol:
+    id: int
+    symbol: str
+    destination_venue: str
+    routable: bool
+    max_quantity_precision: Optional[int] = None
+    max_price_precision: Optional[int] = None
+
+
+@dataclass
+class Cryptocurrency:
+    id: int
+    symbol: str
+    instrument_type: str
+    short_description: str
+    description: str
+    is_closing_only: bool
+    active: bool
+    tick_size: str
+    streamer_symbol: str
+    destination_venue_symbols: List[DestinationVenueSymbol]
+
+
+class Instruments:
+    def get_cryptocurrencies(
+        self, session: Session, symbols: List[str] = None
+    ) -> List[Cryptocurrency]:
+        if symbols:
+            symbol_params = "&".join([f"symbol[]={s}" for s in symbols])
+            url = f"{session.base_url}/instruments/cryptocurrencies?{symbol_params}"
+        else:
+            url = f"{session.base_url}/instruments/cryptocurrencies"
+
+        response = requests.get(url, headers=session.headers)
+        validate_response(response)
+        response_data = json.loads(response.content)
+        cryptocurrencies_data = response_data["data"]["items"]
```

Comment:
A bit nitpicky but the convention in the rest of this repo is single quotes over double, could you do that here?

Reply:
Hmm.. my linter is autocorrecting the single quotes to double qoutes... let me check how I can change it